docs/middlewares.md

```diff
@@ -0,0 +1,38 @@
+# Middlewares
+
+Main goal of middleware is to make IaC and remote resources set to match. For this we filter some default resources, mutate or remove some fields in resources or even create and delete new resources.
```

Comment:
`The main goal of middlewareS is to reconciliate IaC and remote resources. For this we filter default resources, mutate, or remove fields in resources ...`